packages/shared/src/graphql/banner.ts

```diff
@@ -1,10 +1,18 @@
 import { gql } from 'graphql-request';
+import { Theme } from '../components/utilities';
+
+export enum BannerCustomTheme {
+  cabbageOnion = 'cabbage-onion',
+  whitePepper = 'whitePepper',
+}
+
+export type BannerTheme = Theme | BannerCustomTheme;
```

Comment:
AFAIK we dont have a shared type system between the backend and frontend right?

Reply:
You're right